recipes/django-select2/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "django-select2" %}
+{% set version = "7.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9d3330fa0083a03fb69fceb5dcd2e78065cfd08e45c89d4fd727fce4673d3e08
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
under `host` you are going to need `setuptools_scm`, `pytest-runner` and `sphinx` due to the `setup_requires` section in the `setup.cfg`.

Reply:
added. Thanks for the tip